CONTRIBUTING.CHEATSHEET.POSTGRESQL.md

```diff
@@ -0,0 +1,176 @@
+   c22cad586f28   none                            null      local
+   092ed56f9f1b   sequelize-postgres-10-network   bridge    local
+   ```
+   
+   The drivers with _bridge_ are of particular interest. More details can be found using `docker inspect`; for instance, `docker inspect sequelize-postgres-10-network`, reveals the containers bound to it:
+
+   ```json
+      ...
+      "Containers": {
+            "8544cdd984ebca72b8c819ca4195f6d130807fe7a9c2a49cb389926f2c2799d8": {
+                "Name": "sequelize-postgres-10",
+                "EndpointID": "ae0f4df1a94b16b7316d27b086f6e3506a06d692282272e5536762e1053a78f0",
+                "MacAddress": "02:42:ac:16:00:02",
+                "IPv4Address": "172.22.0.2/16",
+                "IPv6Address": ""
+            }
+        },
+      ...
+   ```
+
+   The idea is to have a network bridge that includes both containers. There are two ways to accomplish this:
+
+   1. Add one of the containers to their existing bridges. Using the network we discovered above we can run:
+      ```bash
+      docker network connect sequelize-postgres-10-network pgadmin4
+      ```
+      This will add pgadmin's container (named _pgadmin4_) to the postgres container's network.
+
+   1. Create a new network and add containers. If your preference is to leave the containers in tact, you may want to create a new bridge network and then connect the containers to it. Below we call this network `pg-net`, but it could be named whichever:
+
+      ```bash
+      docker network create  pg-net
+      docker network connect pg-net pgadmin4
+      docker network connect pg-net sequelize-postgres-10
+      ```
+
+   Once the two containers have been bridged, it's time to test out pgadmin:
+      1. Visit `localhost:8888` in the browser and use the email and password set in the command above; `PGADMIN_DEFAULT_EMAIL` and `PGADMIN_DEFAULT_PASSWORD`, respectively.
+      1. Inside pgadmin, use the following credentials to create a new server:
+         ```yaml
+         Host name: sequelize-postgres-10
+         Port: 5432
+         Username/Password: sequelize_test
+         ```
+         Notice: the host can use the container name instead of an IP; if bridged on the pgadmin network, it may require the IP of the database container (found in `docker network inspect <network name>`).
+
+   
+
+1. Running PgAdmin4 as a standalone client (non-containerized) for instance on Windows -> PostgreSQL on WSL2  
+Note: these settings are for the PgAdmin desktop client, not using
+   ```yaml
+   Host: localhost
+   Port: 23010 (or whichever port revealed by `docker ps -a`)
+   Username: sequelize_test
+   Password: sequelize_test
+   ```
+---
+
+## Helpful Notes and Explanations
+1. Most scripts in _package.json_ use `yarn`, which is not a package.json dependency and must installed globally or available by some other means; for instance, it does not need to be specifically installed. Read on:
```

Comment:
I think it would be simpler to link to yarn's install documentation which explains all of this, as otherwise we'll have to keep this in sync with theirs :)

Reply:
Your call.  I didn't realize it until you told me about it :)  Since `yarn` is a dependency of the project (almost all npm scripts call it), but not included in the dev-dependencies, something needs to be stated as to why it wasn't included, or how to use it without installing it.

Linking to the documentation would be a good idea, but still need to be sure the links don't break in the future.